.github/workflows/ci-windows.yml

```diff
@@ -244,3 +244,68 @@ jobs:
           See more details on github.com/DynamoRIO/dynamorio/actions/runs/${{github.run_id}}
         to: dynamorio-devs@googlegroups.com
         from: Github Action CI
+
+  ###########################################################################
+  # 32-bit and 64-bit VS2019 release builds:
```

Comment:
Looks like this is release-build-only so it's skipping a bunch of debug code which might contain warnings or issues in vs2019.  Removing the `nontest_only` below should make it build both; or maybe better to replace `nontest_only` with `debug_only` to cover the most code in one job (else this job might become the slowest one) (we'll get release build covered with the switch in #5195).

Reply:
nit: It still says "release" here

Reply:
I edited it myself.